src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarUrlChrome.java

```diff
@@ -25,7 +33,7 @@
     @BeforeMethod
     public void before() {
         ChromeDriverManager.getInstance().setup();
-        driver = new ChromeDriver();
+        driver = DriverFactory.init(DriverFactory.CHROME);
```

Comment:
Why you are using DriverFactory.init(DriverFactory.FIREFOX); if you have Chrome class?